pyro/distributions/torch/__init__.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import, division, print_function
+
+from pyro.distributions.distribution import Distribution  # noqa: F401
+from pyro.distributions.random_primitive import RandomPrimitive
+from pyro.distributions.torch.beta import Beta
+from pyro.distributions.torch.categorical import Categorical
+from pyro.distributions.torch.dirichlet import Dirichlet
+from pyro.distributions.torch.gamma import Gamma
+from pyro.distributions.torch.normal import Normal
+
+# function aliases
+beta = RandomPrimitive(Beta)
```

Comment:
Do we need to expose these functional forms here too (since they should get initialized in `distributions/__init__.py`)?

Reply:
It's convenient to expose those here for side-by-side testing (as Martin is doing today):
```py
from pyro.distributions import normal as old_normal
from pyro.distributions.torch import normal as new_normal
```

Reply:
or
```py
import pyro.distributions as old_dist
import pyro.distributions.torch as new_dist
```

Reply:
I see, thanks for explaining.